grammar.js

```diff
@@ -489,7 +496,7 @@ module.exports = grammar({
         $.keyword_regproc,
         $.keyword_regtype,
 
-        field("custom_type", $._identifier)
+        field("custom_type", $.object_reference)
```

Comment:
are there any databases in which types belong to schemata? if not `_identifier` is correct but could be aliased

Reply:
It is the case in postgres https://www.postgresql.org/docs/current/sql-createtype.html.

Reply:
interesting, I didn't know that!